yaksh/middleware/user_time_zone.py

```diff
@@ -5,10 +5,10 @@
 
 class TimezoneMiddleware(object):
     """ Middleware to get user's timezone and activate timezone 
-        if user timezone is not available default value 'UTC' is activated """
+        if user timezone is not available default value 'Asia/Kolkata' is activated """
     def process_request(self, request):
         user = request.user
-        user_tz = 'UTC'
+        user_tz = 'Asia/Kolkata'
```

Comment:
Nothing wrong here, but is it really necessary for the default date time value to be Asia/Kolkata? UTC is a time standard, not a time zone. Every timezone is derived wrt UTC(IST being UTC+05:30). It makes more sense for the default value to be UTC.